From e392070a42c59fb1c4c0127f4849ae7f2f982505 Mon Sep 17 00:00:00 2001
From: Terry Lv <r65388@freescale.com>
Date: Mon, 29 Oct 2012 14:18:25 +0800
Subject: [PATCH 2200/2463] ENGR00230538-10: imx6: CAAM: enable secure memory
 and sm test in defconfig

Enable secure memory and sm test in defconfig.

Signed-off-by: Terry Lv <r65388@freescale.com>
---
 arch/arm/configs/imx6_defconfig |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index f62ea0e..55cc89c 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -1068,7 +1068,6 @@ CONFIG_KEYBOARD_GPIO=y
 # CONFIG_KEYBOARD_STOWAWAY is not set
 # CONFIG_KEYBOARD_SUNKBD is not set
 # CONFIG_KEYBOARD_XTKBD is not set
-# CONFIG_KEYBOARD_MXC is not set
 # CONFIG_INPUT_MOUSE is not set
 # CONFIG_INPUT_JOYSTICK is not set
 # CONFIG_INPUT_TABLET is not set
@@ -1640,6 +1639,7 @@ CONFIG_MXC_CAMERA_OV8820_MIPI=m
 CONFIG_MXC_CAMERA_OV5642=m
 CONFIG_MXC_TVIN_ADV7180=m
 CONFIG_MXC_CAMERA_OV5640_MIPI=m
+# CONFIG_MXC_MIPI_CSI2_TVIN_ADV7280 is not set
 CONFIG_MXC_CAMERA_SENSOR_CLK=m
 CONFIG_MXC_IPU_DEVICE_QUEUE_SDC=m
 CONFIG_MXC_IPU_PRP_ENC=m
@@ -1773,7 +1773,6 @@ CONFIG_FB_MXC_TRULY_WVGA_SYNC_PANEL=y
 # CONFIG_FB_MXC_CLAA_WVGA_SYNC_PANEL is not set
 # CONFIG_FB_MXC_SEIKO_WVGA_SYNC_PANEL is not set
 # CONFIG_FB_MXC_SII902X is not set
-# CONFIG_FB_MXC_SII902X_ELCDIF is not set
 # CONFIG_FB_MXC_CH7026 is not set
 # CONFIG_FB_MXC_TVOUT_CH7024 is not set
 # CONFIG_FB_MXC_ASYNC_PANEL is not set
@@ -1856,13 +1855,11 @@ CONFIG_SND_SOC_IMX_SPDIF=y
 CONFIG_SND_SOC_IMX_HDMI=y
 CONFIG_SND_SOC_I2C_AND_SPI=y
 # CONFIG_SND_SOC_ALL_CODECS is not set
-# CONFIG_SND_SOC_WM_HUBS is not set
 CONFIG_SND_SOC_MXC_HDMI=y
 CONFIG_SND_SOC_MXC_SPDIF=y
 CONFIG_SND_SOC_SGTL5000=y
 CONFIG_SND_SOC_CS42888=y
 CONFIG_SND_SOC_WM8962=y
-# CONFIG_SND_SOC_WM8994 is not set
 # CONFIG_SOUND_PRIME is not set
 CONFIG_AC97_BUS=y
 CONFIG_HID_SUPPORT=y
@@ -2698,6 +2695,9 @@ CONFIG_CRYPTO_DEV_FSL_CAAM_CRYPTO_API=y
 CONFIG_CRYPTO_DEV_FSL_CAAM_AHASH_API=y
 CONFIG_CRYPTO_DEV_FSL_CAAM_RNG_API=y
 # CONFIG_CRYPTO_DEV_FSL_CAAM_RNG_TEST is not set
+CONFIG_CRYPTO_DEV_FSL_CAAM_SM=y
+CONFIG_CRYPTO_DEV_FSL_CAAM_SM_SLOTSIZE=7
+CONFIG_CRYPTO_DEV_FSL_CAAM_SM_TEST=y
 # CONFIG_BINARY_PRINTF is not set
 
 #
-- 
1.7.7.4

